orchagent/portsorch.cpp

```diff
@@ -4469,6 +4561,69 @@ void PortsOrch::doVlanMemberTask(Consumer &consumer)
     }
 }
 
+void PortsOrch::doTransceiverInfoTableTask(Consumer &consumer)
```

Comment:
@noaOrMlnx I would suggest to rename "doTransceiverPresenceCheck" since this routine is checking the presence of the module as notified in TRANSCEIVER_INFO SET/DEL